packages/select/src/MenuButton.tsx

```diff
@@ -64,6 +71,8 @@ const MenuButton = React.forwardRef<HTMLElement, Props>(function MenuButton(
   const colorSet = colorSets[mode];
   const sizeSet = sizeSets[size];
 
+  const errorColor = mode === Mode.Light ? uiColors.red.base : '#F97216';
```

Comment:
Should the dark mode color be in the palette?

Reply:
As discussed this will be updated during the reskin